apps/src/applab/designMode.js

```diff
@@ -473,7 +474,8 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging, prefix) {
   if (!Applab.levelHtml) {
     return;
   }
-  var levelDom = $.parseHTML(Applab.levelHtml);
+  var sanitized = sanitizeHtml(Applab.levelHtml);
```

Comment:
Yeah, mostly I just want to make sure that there's some way we discover these errors. Logging to the console we may or may not notice (it might be sufficient), silently sanitizing we would not notice :) Logging a bug to address this in a future commit in some way seems like the right approach :+1: